policyengine_us/parameters/gov/states/nc/ncdhhs/scca/entry/income_rate_by_child_age.yaml

```diff
@@ -0,0 +1,20 @@
+description: North Carolina limits the Subsidized Child Care Assistance program initially to filers with income below this percentage of federal poverty guidelines, based on the age of the dependent.
+metadata:
+  type: single_amount
+  threshold_unit: year
+  amount_unit: /1
+  label: North Carolina Subsidized Child Care Assistance program income limits by child age
+  reference:
+    - title: Subsidized Child Care Assistance Program Policy Manual
+      href: https://policies.ncdhhs.gov/wp-content/uploads/chapter-8-parental-fees-7.pdf#page=8
```

Comment:
```suggestion
      href: https://policies.ncdhhs.gov/wp-content/uploads/chapter-8-parental-fees-7.pdf#page=9
```

Reply:
Why page 9 while it's on page 8?

Reply:
we go off the pdf page numbers not the numbers at the bottom of the page. visiting the link should go to the appropriate page